docs/hint.md

```diff
@@ -1,5 +1,7 @@
 ## Hint
```

Comment:
Bizzarely there's never been any of our many hint example on this page!

Reply:
Seems good to add one